modules/incident-workflowoperation/src/main/java/org/opencastproject/workflow/handler/incident/IncidentCreatorWorkflowOperationHandler.java

```diff
@@ -72,13 +77,18 @@ public WorkflowOperationResult start(WorkflowInstance wi, JobContext ctx) throws
     final WorkflowOperationInstance woi = wi.getCurrentOperation();
     final int code = option(woi.getConfiguration(OPT_CODE)).bind(Strings.toInt).getOrElse(1);
     final Severity severity = option(woi.getConfiguration(OPT_SEVERITY)).bind(parseEnum(Severity.FAILURE)).getOrElse(Severity.INFO);
-    final List<Tuple<String, String>> details =  option(woi.getConfiguration(OPT_DETAILS)).mlist()
-            .bind(splitBy(";")).map(splitBy("="))
-            .filter(Tuples.<String>listHasSize(2)).map(Tuples.<String>fromList()).value();
-    final Map<String, String> params = Immutables.map(
-            option(woi.getConfiguration(OPT_PARAMS)).mlist()
-                    .bind(splitBy(";")).map(splitBy("="))
-                    .filter(Tuples.<String>listHasSize(2)).map(Tuples.<String>fromList()).value());
+
+    final List<Tuple<String, String>> details = Arrays.stream(ArrayUtils.nullToEmpty(
+            StringUtils.split(woi.getConfiguration(OPT_DETAILS), ";")))
+            .map((opt) -> opt.split("="))
+            .filter((t) -> t.length == 2)
+            .map((x) -> Tuple.tuple(x[0], x[1]))
+            .collect(Collectors.toList());
+    final Map<String, String> params = Arrays.stream(ArrayUtils.nullToEmpty(
+            StringUtils.split(woi.getConfiguration(OPT_PARAMS), ";")))
+            .map((opt) -> opt.split("="))
+            .filter((t) -> t.length == 2)
+            .collect(Collectors.toMap(x -> x[0], x -> x[1]));
```

Comment:
Note that there is an extremely subtle difference between `Immutables` and the new code: the new code will fail if there is a duplicate key (because `toMap` checks this), whereas with `Immutables`, the latest key "wins". I'm not sure this is important though.